vscode-extension/src/devcontainer.ts

```diff
@@ -42,37 +42,39 @@ export async function readDevboxJson(workspaceUri: Uri) {
 
 function getDockerfileContent(): String {
     return `
-	# See here for image contents: https://github.com/microsoft/vscode-dev-containers/tree/v0.245.2/containers/debian/.devcontainer/base.Dockerfile
-
-	# [Choice] Debian version (use bullseye on local arm64/Apple Silicon): bullseye, buster
-	ARG VARIANT="buster"
-	FROM mcr.microsoft.com/vscode/devcontainers/base:0-\${VARIANT}
-
-	# These dependencies are required by Nix.
-	RUN apt update -y
-	RUN apt -y install --no-install-recommends curl xz-utils
-
-	USER vscode
-
-	# Install nix
-	ARG NIX_INSTALL_SCRIPT=https://nixos.org/nix/install
-	RUN curl -fsSL \${NIX_INSTALL_SCRIPT} | sh -s -- --no-daemon
+    FROM alpine:3
+
+    # Setting up devbox user
```

Comment:
Reuse the same `.tmpl` file instead of copy pasting the string?

Reply:
fair point! 
I updated the code to use the CLI similar to the rest of commands in the extension. It's very simple now.